source/_components/sensor.rmvtransport.markdown

```diff
@@ -60,7 +60,12 @@ max_journeys:
   description: Specify the maximal number of journeys.
```

Comment:
The nesting of the config schema is missing and the `next_departure` key.

Reply:
I hope this is fixed now?